core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/commandqueue/commands/CancelTransferCommand.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.dataspaceconnector.transfer.core.commandqueue.commands;
+
+import org.eclipse.dataspaceconnector.spi.transfer.command.Command;
+
+public class CancelTransferCommand extends Command {
```

Comment:
We could define a `TransferProcessCommand` abstraction, that already contains the `transferProcessId`

Reply:
thought of that too. Might be worth the effort. That could also handle the "TP-not-found-case"